card/spec/lib/card/format_spec.rb

```diff
@@ -6,26 +6,26 @@
     let(:format) { described_class.new Card.new }
 
     it "should respect defaults" do
-      expect(format.show_view?( :menu, :default_visibility=>:show )).to be_truthy
-      expect(format.show_view?( :menu, :default_visibility=>:hide )).to be_falsey
+      expect(format.show_view?( :menu, default_visibility: :show )).to be_truthy
+      expect(format.show_view?( :menu, default_visibility: :hide )).to be_falsey
       expect(format.show_view?( :menu, {}                          )).to be_truthy
     end
 
     it "should respect developer default overrides" do
-      expect(format.show_view?( :menu, :optional_menu=>:show, :default_visibility=>:hide )).to be_truthy
-      expect(format.show_view?( :menu, :optional_menu=>:hide, :default_visibility=>:show )).to be_falsey
-      expect(format.show_view?( :menu, :optional_menu=>:hide                             )).to be_falsey
+      expect(format.show_view?( :menu, optional_menu: :show, default_visibility: :hide )).to be_truthy
+      expect(format.show_view?( :menu, optional_menu: :hide, default_visibility: :show )).to be_falsey
+      expect(format.show_view?( :menu, optional_menu: :hide                             )).to be_falsey
     end
 
     it "should handle args from inclusions" do
-      expect(format.show_view?( :menu, :show=>'menu', :default_visibility=>:hide         )).to be_truthy
-      expect(format.show_view?( :menu, :hide=>'menu, paging', :default_visibility=>:show )).to be_falsey
-      expect(format.show_view?( :menu, :show=>'menu', :optional_menu=>:hide              )).to be_truthy
+      expect(format.show_view?( :menu, show: 'menu', default_visibility: :hide         )).to be_truthy
```

Comment:
Line is too long. [102/80]<br>Unnecessary spacing detected.<br>Space inside parentheses detected.